src/dispatch/incident/flows.py

```diff
@@ -650,6 +659,10 @@ def incident_create_flow(*, incident_id: int, checkpoint: str = None, db_session
     set_conversation_topic(incident)
 
     # we update the incident ticket
+    incident_type_data = incident_type_service.get_by_name(
+        db_session=db_session, name=incident.incident_type.name
+    )
+
```

Comment:
Can we move it inside `update_incident_ticket`, so we don't have to add this code to all the places where we call `update_incident_ticket`? Same for when we call `create_incident_ticket`.

Reply:
updated. Had to add passing in the incident type name in a few places.